features/polyExplorer/src/screens/storiesInfoScreens/barChartInfo/companiesBarChartInfo.jsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+
+import i18n from "../../../i18n.js";
+import StoriesInfoScreen from "../../../components/clusterStories/storiesInfoScreen.jsx";
+// import Infographic from "../../../components/infographic/infographic.jsx";
+
+const CompaniesBarChartInfo = () => {
+    const CompaniesBarChartInfoContent = [
+        <div className="base-info-padding">
+            <p>{i18n.t("companiesBarChartInfoScreen:p1")}</p>
+            {/* <Infographic
+                type="verticalBarChartInfo"
+                texts={{
+                    text1: i18n.t("infographic:verticalBarChartInfo.text1"),
+                    boldLabel: i18n.t(
+                        "infographic:verticalBarChartInfo.boldLegend"
+                    ),
+                    textYaxis: i18n.t(
+                        "infographic:verticalBarChartInfo.textYaxis"
+                    ),
+                    textXaxis: i18n.t(
+                        "infographic:verticalBarChartInfo.textXaxis"
+                    ),
+                }}
+            /> */}
```

Comment:
Whats up here?

Reply:
This infographic is crashing the app. I'll talk to the designers because it seems that the issue could be in Figma.

Reply:
Can take a look together if you'd like